docs/docs/creating-a-source-plugin.md

```diff
@@ -2,62 +2,29 @@
-  Gatsby calls [nodes](/docs/node-interface/).
-- **Transformer** plugins "transform" data provided by source plugins into new
-  nodes and/or node fields.
+These systems can be CMSs like Contentful or Wordpress, other cloud services like Lever and Strava, or your local filesystem -- literally anything that has an API. Currently, Gatsby has [over 300 source plugins](/plugins/?=gatsby-source).
 
-This doc focuses on source plugins and uses `gatsby-source-filesystem` to explain how source plugins work.
+Once a source plugin brings data into Gatsby's system, it can be transformed further with **transformer plugins**.
 
 ## What do source plugins do?
 
-The [`gatsby-source-filesystem`](/packages/gatsby-source-filesystem/) plugin
-"sources" data about files from the file system. It creates nodes with a type
-`File`, each File node corresponding to a file on the filesystem. On each node
-are fields like the `absolutePath`, `extension`, `modifiedTime`, etc.
+At a high-level, a source plugin:
 
-## What fields are required?
-
-### `media type` is not required, yet necessary to work with transformer plugins
-
-Each node created by the filesystem source plugin includes the
-raw content of the file and its _media type_.
-
-[A **media type**](https://en.wikipedia.org/wiki/Media_type) (also **MIME type**
-and **content type**) are an official way to identify the format of
-files/content that is transmitted on the internet e.g. over HTTP or through
-email. You're probably familiar with many media types such as
-`application/javascript`, `application/pdf`, `audio/mpeg`, `text/html`,
-`text/plain`, `image/jpeg`, etc.
-
-Each source plugin is responsible for setting the media type for the nodes they
-create. This way, source and transformer plugins can work together easily.
-
-This is not a required field but it's the way for source plugins to indicate to
-transformers that there is "raw" data that can still be further processed. It
-allows plugins to remain small and focused. Source plugins don't have to have
-opinions on how to transform their data. They can just set the `mediaType` and
-push that responsibility to transformer plugins.
-
-For example, it's quite common for services to allow you to add content as
-markdown. If you pull that markdown into Gatsby and create a new node, what
-then? How would a user of your source plugin convert that markdown into HTML
-they can use in their site? Luckily you don't have to do anything. Just create a
-node for the markdown content and set its mediaType as `text/markdown` and the
-various Gatsby markdown transformer plugins will see your node and transform it
-into HTML.
-
-This loose coupling between the data source and the transformer plugins allow Gatsby site builders to quickly assemble complex data transformation pipelines with
-little work on their (and your (the source plugin author)) part.
+- Ensure local data is synced with its source and 100% accurate.
```

Comment:
```suggestion
- Ensures local data is synced with its source and is 100% accurate.
```

this just makes these bullets read from the sentence fragment above it a little smoother.